packages/oidc-client/src/lib/client.store.ts

```diff
@@ -131,16 +161,32 @@ export async function oidc<ActionType extends ActionTypes = ActionTypes>({
         }
       },
     },
+    /**
+     * An object containing methods for token exchange
+     */
     token: {
-      exchange: async (code: string, state: string, options?: Partial<StorageConfig>) => {
+      /**
+       * @function exchange
+       * @description Exchanges an authorization code for tokens using the token endpoint from the wellknown configuration
+       *              and stores them in the configured storage.
+       * @param code - The authorization code received from the authorization server.
+       * @param state - The state parameter from the authorization URL creation.
+       * @param options - Optional storage configuration for persisting tokens.
+       * @returns {Promise<TokenExchangeErrorResponse | OauthTokens>}
```

Comment:
Looks like the return type here should be updated to `Promise<TokenExchangeResponse | GenericError | TokenExchangeErrorResponse>`